tests/50federation/31room-send.pl

```diff
@@ -145,3 +146,71 @@
          });
       });
    };
+
+test "Ephemeral messages received from servers are correctly expired",
+   requires => [ local_user_and_room_fixtures(), federation_user_id_fixture(),
+                 $main::OUTBOUND_CLIENT ],
+
+   do => sub {
+      my ( $local_user, $room_id, $federated_user, $outbound_client ) = @_;
+
+      my $now_ms = int( time() * 1000 );
+      my $filter = '{"types":["m.room.message"]}';
+
+      $outbound_client->join_room(
```

Comment:
`federated_rooms_fixture` is a useful way to cut out some of this boilerplate fyi. no real need to change this unless you want to.